app/backend/src/couchers/servicers/admin.py

```diff
@@ -802,19 +661,3 @@ def AccessStats(self, request, context, session):
             )
 
         return out
-
-    def SendBlogPostNotification(self, request, context, session):
-        if len(request.title) > 50:
-            context.abort_with_error_code(grpc.StatusCode.FAILED_PRECONDITION, "admin_blog_title_too_long")
-        if len(request.blurb) > 100:
-            context.abort_with_error_code(grpc.StatusCode.FAILED_PRECONDITION, "admin_blog_blurb_too_long")
-        queue_job(
-            session,
-            "generate_new_blog_post_notifications",
-            payload=jobs_pb2.GenerateNewBlogPostNotificationsPayload(
-                url=request.url,
-                title=request.title,
-                blurb=request.blurb,
-            ),
-        )
-        return empty_pb2.Empty()
```

Comment:
We have to deploy this in concert with FE changes, otherwise there'll be errors. Should we maybe keep the old handlers until FE is deployed?